2021-02-19 03:12:38.200053: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer.so.6'; dlerror: libnvinfer.so.6: cannot open shared object file: No such file or directory
2021-02-19 03:12:38.200178: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer_plugin.so.6'; dlerror: libnvinfer_plugin.so.6: cannot open shared object file: No such file or directory
2021-02-19 03:12:38.200195: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:30] Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
2021-02-19 03:12:40.679994: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2021-02-19 03:12:40.680220: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2021-02-19 03:12:40.680269: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (c346.oscer.ou.edu): /proc/driver/nvidia/version does not exist
2021-02-19 03:12:40.680505: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2021-02-19 03:12:40.702740: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2297400000 Hz
2021-02-19 03:12:40.703730: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x43db9e0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-19 03:12:40.703757: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
Traceback (most recent call last):
  File "nn.py", line 207, in <module>
    execute_exp(args)
  File "nn.py", line 67, in execute_exp
    activation1=args.activation1,activation2=args.activation2, lrate=args.lrate)
  File "nn.py", line 46, in build_model
    epsilon=None, decay=0.0, amsgrad=False)
  File "/home/fagg/pythonenv/tensorflow/lib/python3.6/site-packages/tensorflow_core/python/keras/optimizer_v2/adam.py", line 137, in __init__
    super(Adam, self).__init__(name, **kwargs)
  File "/home/fagg/pythonenv/tensorflow/lib/python3.6/site-packages/tensorflow_core/python/keras/optimizer_v2/optimizer_v2.py", line 258, in __init__
    if kwargs[k] < 0:
TypeError: '<' not supported between instances of 'str' and 'int'
